cfbs/commands.py

```diff
@@ -965,6 +958,7 @@ def _print_module_info(data):
             print("{}: {}".format(key.title().replace("_", " "), value))
 
 
+@cfbs_command("show")
```

Comment:
I'd squash this commit with the one introducing the decorator.

Reply:
Will do!